function-template/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG IMAGE
+FROM ${IMAGE}
+
+ARG FUNCTION_SRC=function.txt
+
+COPY ${FUNCTION_SRC} .
+
+# Package name
+RUN while read -r _ a _; do echo "$a" ; done <<< $(cat ${FUNCTION_SRC} | grep "^[ $(printf '\t')]*package" | head -1) | sed s'/;*$//' > /tmp/package
+
+# Class name
+RUN while read -r _ _ a _; do echo "$a" ; done <<< $(cat ${FUNCTION_SRC} | sed -e "s/static//" | grep "^[ $(printf '\t')]*public[ $(printf '\t')]*class" | head -1) > /tmp/class
+
+# Copy source file
+RUN mkdir -p src/main/java/$(cat /tmp/package | sed 's:\.:\/:g')
+RUN mv ${FUNCTION_SRC} src/main/java/$(cat /tmp/package | sed 's:\.:\/:g')/$(cat /tmp/class).java
```

Comment:
Could you make a variable to store `src/main/java/$(cat /tmp/package | sed 's:\.:\/:g')` and then reuse that in these two statements?

Reply:
From what I know you can't store the result of the command into a variable in a Dockerfile but I could store it in a temp file similar to class and package and reuse it then

Reply:
Then I wouldn't worry about it.